### ISSUE TITLE
[Feature request] Create the title parameter for the angular axis of polar charts

### ISSUE THREAD

**Issue by AAnzel** [plotly/plotly.js#6332]
As the title says, the title of the angular axis can't be changed or set as a parameter. I can change the radial axis title, but not the angular one. There is also no parameter in the documentation [https://plotly.com/python/reference/layout/polar/#layout-polar-angularaxis](https://plotly.com/python/reference/layout/polar/#layout-polar-angularaxis). The code for the redefinition of my chart is below.

```
chart_result.update_layout(
        polar = dict(
            sector=[0, 180],
            angularaxis=dict(
                #thetaunit = "degrees",
                dtick=10,
                #rotation=270,
                direction="counterclockwise",
                #tickmode="array",
                tickvals=list_angular_ticks,
                ticktext=list_angular_labels,
            ),
        ),
        height=600
    )
    
    chart_result.update_polars(
        radialaxis_title_text='Normalized STD',
    )
```

I also have a proposal for the placement of the title. As seen in the code below, I will only consider the upper half of a circle. Therefore, the title could be placed in the middle - at 90 degrees (above the tick labels). This is also valid if we have, for example, only one quadrant (0-90). In that case, the label should be placed above 45 degrees tick label. An example can be seen below. This logic also works for other domains.
![image](https://user-images.githubusercontent.com/44969003/193030700-f1ff94da-df8a-44b3-9b22-a048baab3891.png)

**Comment by Jdruwe** [comment]
Any update on this?

**Comment by emilykl** [comment]
@AAnzel @Jdruwe Apologies that this issue didn't get a timely response. The plotly team likely won't have bandwidth in the near future, but happy to discuss implementation here as a first step. It's a reasonable feature request. I agree that the question of where to place the axis label is the most difficult question.

I think your proposal is reasonable @AAnzel, although it's less clear in the case of a radial plot which contains the full 360°. I suppose we could have a `title.angle` parameter which controls the angular position of the title.

@alexcjohnson curious if you have any thoughts on this one!

**Comment by alexcjohnson** [comment]
I agree, this proposal looks great for any incomplete angular axis. If it makes a full circle it feels like the ideal solution is a bit more nuanced: A linear axis that's complete will generally contain zero, either as its start or end or somewhere in the middle, so we could put the title there. If it's a category axis or linear that doesn't contain zero I'd probably put it at the bottom. Either way we probably want the ability to move it to any angle, as no decision we make will be able to capture every user's preferences in all situations.
